jobspec/test-fixtures/tg-service-check-body.hcl

```diff
@@ -0,0 +1,28 @@
+job "group_service_check_script" {
```

Comment:
We're not adding any new fields to HCL1, let's move this into the `jobspec2` test fixtures.